### PR TITLE
js syntax

### PR DESCRIPTION
**Pull request by lalitsaud1997**
js syntax

### CODE REVIEW

File: js_syntax.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JS_Syntax</title>
+</head>
+<body>
```
Comment (lalitsaud1997) [inline js_syntax.html:9]: js syntax

File: js_syntax.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <!-- JavaScript syntax is the set of rules, how JavaScript programs are constructed: -->
+    // How to create variables:
```
Comment (saudG1997) [inline js_syntax.html:11]: variable created

File: js_syntax.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    var x;
+    let y;
+
+    // How to use variables:
```
Comment (saudG1997) [inline js_syntax.html:15]: variable use

**Review by saudG1997 — APPROVED** [review]

**Review by lalitsaud1997 — COMMENTED** [review]
lalit

**Pull request merged**